### PR TITLE
chore: bump deps

### PR DESCRIPTION
**Pull request by lino-levan**

### CODE REVIEW

**Comment by lino-levan** [comment]
@kt3k Sorry for the ping. Please take a look if you can!

**Review by kt3k — APPROVED** [review]
LGTM

**Comment by kt3k** [comment]
@satyarohith Can you also take a look?

File: src/utils/api.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import { LineStream } from "https://deno.land/std@0.116.0/streams/delimiter.ts";
+import { TextLineStream } from "https://deno.land/std@0.170.0/streams/mod.ts";
```
Comment (satyarohith) [inline src/utils/api.ts:1]:
```suggestion
import { TextLineStream } from "../../deps.ts";
```

**Review by satyarohith — APPROVED** [review]
LGTM - thanks for the contribution!

**Pull request merged**